src/config/Config.coffee

```diff
@@ -92,6 +92,14 @@ Config =
         false
         'Add buttons to navigate to top / bottom of thread.'
       ]
+      'Unique ID Navigation': [
+        false
+        'Add buttons to navigate to posts having the same unique ID.'
+      ]
+      'Capcode Navigation': [
+        false
+        'Add buttons to navigate to posts having the same capcode.'
+      ]
```

Comment:
There are already a lot of options; we should find out whether there's anyone who wants one of these but not the other.